robottelo/ui/contentviews.py

```diff
@@ -95,17 +95,18 @@ def delete_version(self, name, version):
         self.click(locators['contentviews.next_button'])
         self.click(locators['contentviews.confirm_remove_ver'])
 
-    def search(self, element_name):
+    def search(self, element_name, element_locator=None, search_key=None,
+               katello=True, button_timeout=15, result_timeout=15):
         """Uses the search box to locate an element from a list of elements"""
-        element = None
-        strategy, value = locators['contentviews.key_name']
-        searchbox = self.wait_until_element(common_locators['kt_search'])
-        if searchbox:
-            searchbox.clear()
-            searchbox.send_keys(escape_search(element_name))
-            self.click(common_locators['kt_search_button'])
-            element = self.wait_until_element((strategy, value % element_name))
-        return element
+        Navigator(self.browser).go_to_content_views()
+        return super(ContentViews, self).search(
+            element_name,
+            locators['contentviews.key_name'] or element_locator,
+            search_key=search_key,
+            katello=katello,
+            button_timeout=button_timeout,
+            result_timeout=result_timeout,
+        )
```

Comment:
I believe 'search_key' is only required for the user entity which would be "login". For others it's not required by default.